modules/context/repo.go

```diff
@@ -1082,3 +1098,54 @@ func (ctx *Context) IssueTemplatesErrorsFromDefaultBranch() ([]*api.IssueTemplat
 	}
 	return issueTemplates, invalidFiles
 }
+
+func ExtractIssueConfigFromYaml(configContent []byte) (api.IssueConfig, error) {
+	config := api.IssueConfig{
+		BlankIssuesEnabled: true,
+	}
+
+	err := yaml.Unmarshal(configContent, &config)
+	if err != nil {
+		return api.IssueConfig{}, err
+	}
+
+	return config, nil
+}
+
+func (ctx *Context) IssueConfigFromDefaultBranch() (api.IssueConfig, error) {
+	defaultIssueConfig := api.IssueConfig{
+		BlankIssuesEnabled: true,
+	}
+
+	commit, err := ctx.Repo.GitRepo.GetBranchCommit(ctx.Repo.Repository.DefaultBranch)
+	if err != nil {
```

Comment:
Why not return err?

Reply:
I think, if anything goes wrong here, three is nothing a normal User can do. So we should not show it in the GUI.

Reply:
Imo, the error should be posted to UI since we cannot read default branch.